samples/ComputerVision/DomainSpecificContent/Program.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.CognitiveServices.Samples.ComputerVision.DomainSpecificContent
+{
+    using Microsoft.Azure.CognitiveServices.Vision.ComputerVision;
+    using Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Models;
+
+    class Program
+    {
+        public const string subscriptionKey = "<your training key here>"; //Insert your Cognitive Services subscription key here
+        public const string endpoint = "https://westus.api.cognitive.microsoft.com"; // You must use the same Azure region that you generated your subscription keys for.  Free trial subscription keys are generated in the westus region. 
+
+        static void Main(string[] args)
+        {
+            try
+            {
+                DomainSpecificContentSample.RunAsync(endpoint, subscriptionKey).Wait(5000);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
+            }
+
+            Console.WriteLine("\nPress ENTER to exit.");
+            Console.ReadLine();
+        }
+    }
+    public class DomainSpecificContentSample
+    {
+        public static async Task RunAsync(string endpoint, string key)
+        {
+            ComputerVisionClient computerVision = new ComputerVisionClient(new ApiKeyServiceClientCredentials(key))
+            {
+                Endpoint = endpoint
+            };
+
+            string localImagePath = @"Images\celebrities.jpg";   // See this repo's readme.md for info on how to get these images. Alternatively, you can just set the path to any appropriate image on your machine.
+            string remoteImageUrl = "https://github.com/Azure-Samples/cognitive-services-sample-data-files/raw/master/ComputerVision/Images/landmark.jpg";
+
+            Console.WriteLine("Domain Specific Content being analyzed ...");
+            await DomainSpecificContentFromUrlAsync(computerVision, remoteImageUrl);
+            await DomainSpecificContentFromStreamAsync(computerVision, localImagePath);
+        }
+
+        // Analyze a remote image
+        private static async Task DomainSpecificContentFromUrlAsync(ComputerVisionClient computerVision, string imageUrl)
+        {
+            if (!Uri.IsWellFormedUriString(imageUrl, UriKind.Absolute))
+            {
+                Console.WriteLine("\nInvalid remote image url:\n{0} \n", imageUrl);
+                return;
+            }
+            
+            DomainModelResults analysis = await computerVision.AnalyzeImageByDomainAsync("landmarks",imageUrl);  //change the first parameter to "landmarks" if that is the domain you are interested in
+
+            Console.WriteLine(imageUrl);
+            DisplayResults(analysis); 
+        }
+
+        // Analyze a local image
+        private static async Task DomainSpecificContentFromStreamAsync(ComputerVisionClient computerVision, string imagePath)
```

Comment:
Add a required string parameter for the domain name, so the caller can pass it in and turn this into a more generic method.